src/DynamoCoreWpf/Views/PackageManager/PackagePathView.xaml

```diff
@@ -1,242 +1,141 @@
-                                </DataTrigger>
-                            </ControlTemplate.Triggers>
-
-                        </ControlTemplate>
-                    </Setter.Value>
-                </Setter>
-            </Style>
-
+          
+            <DataTemplate x:Key="packageSearchPathItemTemplate">
+             
+                <Grid ToolTip="{Binding}">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto"></ColumnDefinition>
+                        <ColumnDefinition Width="Auto"></ColumnDefinition>
+                        <ColumnDefinition MaxWidth="250"></ColumnDefinition>
+                        <ColumnDefinition Width="Auto"></ColumnDefinition>
+                        <ColumnDefinition Width="Auto"></ColumnDefinition>
+                    </Grid.ColumnDefinitions>
+                    <!--the up arrow button-->
+                    <Button
+                         Grid.Column="0"
+                        Margin="7,7,3,7"
+                              Command="{Binding Path=DataContext.MovePathUpCommand,
+          RelativeSource={RelativeSource AncestorType={x:Type UserControl}}}"
+                         CommandParameter="{Binding}"
+                        ToolTip="{x:Static p:Resources.PackagePathViewToolTipUp}"
+                        Background="{StaticResource PreferencesWindowBackgroundColor}"
+                        Style="{StaticResource FlatIconButtonStyle}">
+                        <Image Source="/DynamoCoreWpf;component/UI/Images/PackageManager/up_16_16.png" Width="15" Height="15"></Image>
+                    </Button>
+                    <!--the down arrow button-->
+                    <Button
+                         Grid.Column="1"
+                         Margin="3,7,7,7"
+                              Command="{Binding Path=DataContext.MovePathDownCommand,
+          RelativeSource={RelativeSource AncestorType={x:Type UserControl}}}"
+                         CommandParameter="{Binding}"
+                         ToolTip="{x:Static p:Resources.PackagePathViewToolTipDown}"
+                        Background="{StaticResource PreferencesWindowBackgroundColor}"
+                         Style="{StaticResource FlatIconButtonStyle}">
+                        <Image Source="/DynamoCoreWpf;component/UI/Images/PackageManager/down_16_16.png" Width="15" Height="15"></Image>
+                    </Button>
+                    <!--the path text-->
+                    <TextBlock x:Name="PathTextBlock" Text="{Binding}"
+                                Grid.Column="2"
+                                Margin="7,4,7,4"
+                                Background="{StaticResource PreferencesWindowItemDarkerBackgroundColor}"
+                                MinHeight="24"
+                                Padding="5,5,5,5"
+                                TextTrimming="CharacterEllipsis"
+                                FontSize="12"
```

Comment:
Is this font size re-usable ? I wonder if you can turn it into a style ...

Reply:
@pinzart90 I'n not aware of anyway to break out the font specific information for a style, we could have a style for text blocks I guess - but I'm not sure it will be useful as they are pretty specific to each element.

Reply:
added a new style for this with font size and text trimming behavior.